plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/BuildMojo.java

```diff
@@ -64,8 +66,12 @@
     @Parameter
     private ProcessorConfig generator;
 
+    @Parameter(property = "fabric8.build.jib", defaultValue = "false")
+    private boolean isJib;
+
     /**
-     * Enrichers used for enricher build objects
+     * Enrichers used for enri @Parameter(property = "fabric8.to", defaultValue = "")
```

Comment:
This doesn't look right, maybe you commented some parameter?

Reply:
No this is not needed , I will replace it!